mempool/src/counters.rs

```diff
@@ -307,9 +307,10 @@ pub static CORE_MEMPOOL_GC_LATENCY: Lazy<HistogramVec> = Lazy::new(|| {
 });
 
 pub static CORE_MEMPOOL_TXN_CONSENSUS_PULLED: Lazy<Histogram> = Lazy::new(|| {
-    register_histogram!(
+    register_histogram_vec!(
```

Comment:
Suggest renaming the metric too. No idea how our TSDB would react to this migration.

Reply:
Changed the name.